cocotb/share/makefiles/simulators/Makefile.ghdl

```diff
@@ -26,15 +26,12 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ###############################################################################
 
-include $(shell cocotb-config --makefiles)/Makefile.inc
+TOPLEVEL_LANG ?= vhdl
 
-ifneq ($(VERILOG_SOURCES),)
+include $(shell cocotb-config --makefiles)/Makefile.inc
 
-$(COCOTB_RESULTS_FILE):
-	@echo "Skipping simulation as Verilog is not supported on simulator=$(SIM)"
-clean::
-
-else
+ifeq ($(VERILOG_SOURCES),)
+ifeq ($(TOPLEVEL_LANG),vhdl)
```

Comment:
```suggestion
ifneq ($(and $(VHDL_SOURCES),$(filter $(TOPLEVEL_LANG),vhdl)),)
```
Something like this? Does not look great but avoids repeating code.

Reply:
Do we even care about the value of sources? GHDL, Verilator, and Icarus will just ignore the value of `*_SOURCES` for the language it doesn't support.

Reply:
I don't even think we need the `*_SOURCES` check.